File: 3617-find-the-original-typed-string-i/find-the-original-typed-string-i.py
class Solution:
    def possibleStringCount(self, word: str) -> int:
        n=len(word)
        counts=0
        result=1
        for i in range(n):
            counts+=1
            if i+1==n or (i+1<n and word[i]!=word[i+1]):
                result+=(counts-1)
                counts=0
        return result

File: 3617-find-the-original-typed-string-i/find-the-original-typed-string-i.java
class Solution {
    public int possibleStringCount(String word) {
        int result=1;
        int n=word.length();
        int prev=0;
        for(int i=0;i<n;i++){
            prev+=1;
            if(i==n-1 || (word.charAt(i)!=word.charAt(i+1))){
                result+=(prev-1);
                prev=0;
            }
        }
        return result;

        
    }
}